### DISCUSSION TITLE
2005/12/29/%E5%9B%9E%E9%A1%BE2005%E5%9B%BE%E6%96%87%E8%AE%B0%E5%BD%95

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#70] [Announcements]
# 2005/12/29/%E5%9B%9E%E9%A1%BE2005%E5%9B%BE%E6%96%87%E8%AE%B0%E5%BD%95

难怪有些人对网络痛恨的不得了，拼命的想封杀网络。有了网络，信息传递的太快了，刚干了点丢人的事，一转眼全世界都知道了。瞧瞧这个2005年丢人辑录：

https://blog.qizhen.xyz/2005/12/29/%E5%9B%9E%E9%A1%BE2005%E5%9B%BE%E6%96%87%E8%AE%B0%E5%BD%95

**Comment by ruanqizhen** [comment]
之前博客下的留言：

花朵旖旎
2005年12月30日 09:52 编辑
说实话以前没有仔细看你的space（惭愧ing）;P今天仔细看看发现Qizhen真是一个很性情的人啊，呵呵！

回复
人淡如菊
2005年12月29日 09:58 编辑
哈哈
